rotkehlchen/accounting/accountant.py

```diff
@@ -71,6 +71,7 @@ def __init__(
         # map event rules signatures to a list of event identifiers affected by them
         # used to know which events need to be invalidated when updating a rule
         self.processable_events_cache_signatures: DefaultLRUCache[int, list[int]] = DefaultLRUCache(default_factory=list, maxsize=PROCESSABLE_EVENTS_CACHE_SIZE)  # noqa: E501
+        self.ignored_asset_ids: set[str] = set()  # populated in process_history
```

Comment:
```suggestion
        self.ignored_asset_ids: set[str] = set()  # populated in process_history so that we load them in memory once during accounting and not reload them from the DB for every single event processing  # noqa: E501
```